compiler/luci-micro/standalone/CMakeLists.txt

```diff
@@ -7,6 +7,16 @@ set(BUILD_WHITELIST "dummy")
 add_subdirectory(${NNAS_ROOT}/infra/nncc ${CMAKE_CURRENT_BINARY_DIR}/nncc)
 
 set(ONE_COMPILER_SRC_DIR "${NNAS_PROJECT_SOURCE_DIR}/compiler")
+
+# Fixing EXT_OVERLAY_DIR to the correct path to the overlay dir
+# due to CMAKE_CURRENT_BINARY_DIR points to .../compiler/luci-micro/standalone_arm dir
+# and that is why EXT_OVERLAY_DIR points to .../compiler/luci-micro/standalone_arm/overlay
+# but correct path is .../overlay
+get_filename_component(PARENT_DIR ${CMAKE_CURRENT_BINARY_DIR} DIRECTORY)
+get_filename_component(PARENT_DIR ${PARENT_DIR} DIRECTORY)
+get_filename_component(PARENT_DIR ${PARENT_DIR} DIRECTORY)
+set(EXT_OVERLAY_DIR "${PARENT_DIR}/overlay")
```

Comment:
maybe you can use `NNCC_OVERLAY_DIR` ?

Reply:
`NNCC_OVERLAY_DIR` also points to `.../compiler/luci-micro/standalone_arm/overlay`
Actually, `EXT_OVERLAY_DIR` is a copy of `NNCC_OVERLAY_DIR`:
https://github.com/Samsung/ONE/blob/1116490d69a755822dee3b60c040a4934f3450f5/infra/nncc/CMakeLists.txt#L36

Reply:
Q) why do you declare `EXT_OVERLAY_DIR` complicated like this not using `NNCC_OVERLAY_DIR`?
Is this something you have in mind to extract `luci-micro` out from `compiler` folder and put in root?

Reply:
> Q) why do you declare `EXT_OVERLAY_DIR` complicated like this not using `NNCC_OVERLAY_DIR`? Is this something you have in mind to extract `luci-micro` out from `compiler` folder and put in root?

Because `NNCC_OVERLAY_DIR` points to `some_build_dir/compiler/luci-micro/standalone_arm/overlay`, but actually it should point to `some_build_dir/overlay`. if we do not redefine the `EXT_OVERLAY_DIR`, then an error will occur when trying to execute the `mio-circle04`:
https://github.com/Samsung/ONE/blob/1116490d69a755822dee3b60c040a4934f3450f5/compiler/luci-micro/standalone/CMakeLists.txt#L24

Reply:
> Because NNCC_OVERLAY_DIR points to some_build_dir/compiler/luci-micro/standalone_arm/overlay

This is strange. I guess `luci-micro/standalone_arm` is not using default CMake as others.
I hope you fix this strange behaviour because this differs from other modules and may confuse other. Adding a NOTE can help understanding why `EXT_OVERLAY_DIR` is defined here too.

Reply:
@seanshpark 
> I guess luci-micro/standalone_arm is not using default CMake as others.

Yes, we can not use common cmake infrastructure, because luci-micro uses `arm` compiler, and other parts (like `flatc` from externals) are built with x86 (host) compiler.

@BalyshevArtem 
I've investigated what happening here, and found this:

everything works, but take a look:
- `flatbuffers` package is not found
- `mio-cirlce` is not built because of the previous item
- `luci-interpreter` for MCU successfully built, because `mio-circle` and `luci-import` are not used anywhere in interpreter targets and not actually built

In conclusion:
This change **does not** fix actual build, but **sets essential definitions** for common cmake scripts, so they can function as designed.

We can discuss this in detail offline

Reply:
Please merge this without my approval.

Reply:
@binarman, I have updated this cmake, now `flatbuffers` package can be found and `mio-circle` can be build